components/dashboard/DashboardWelcomeScreen.tsx

```
@@ -23,14 +44,21 @@ export const DashboardWelcomeScreen = () => {
         </div>
         
         <div className="flex flex-col sm:flex-row gap-3 md:gap-4 w-full max-w-md">
-          <button className="py-3 md:py-4 px-6 md:px-10 bg-primary text-white rounded-lg hover:bg-primary-dark transition-colors duration-200 font-medium text-sm md:text-base flex-1">
+          <button className="py-3 md:py-4 px-6 md:px-10 bg-primary text-white rounded-lg hover:bg-primary-dark transition-colors duration-200 font-medium text-sm md:text-base flex-1 hover:cursor-pointer"
+          onClick={() => setShowCreateTaskForm(true)}
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)